apps/web/src/ee/clerk/providers/EnterpriseAuthProvider.tsx

```diff
@@ -102,11 +102,11 @@ export const EnterpriseAuthProvider = ({ children }: { children: React.ReactNode
       if (hasOrgs) {
         const firstOrg = clerkUser.organizationMemberships[0].organization;
         setActive({ organization: firstOrg });
-      } else {
-        navigate(ROUTES.AUTH_SIGNUP_ORGANIZATION_LIST);
```

Comment:
this piece of code was getting hit, but for some reason it wasn't doing the navigate, I have no idea why :/